ressourcen-profile/Profile-observation-VitalSignDE_GCS.StructureDefinition.xml

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns='http://hl7.org/fhir'>
+  <id value='observation-de-vitalsign-gcs'/>
+  <url value='http://fhir.de/StructureDefinition/observation-de-vitalsign-gcs'/>
+  <name value='VitalSignDE_GCS'/>
+  <version value="1.0.0"/>
+  <status value='draft'/>
+  <date value='2021-11-04'/>
+  <publisher value='HL7 Deutschland e.V. (Technisches Komitee FHIR)'/>
+  <contact>
+    <telecom>
+      <system value='url'/>
+      <value value='http://hl7.de/technische-komitees/fhir/'/>
+    </telecom>
+  </contact>
+  <copyright value='HL7 Deutschland e.V.'/>
+  <fhirVersion value='4.0.1'/>
+  <mapping>
+    <identity value='workflow'/>
+    <uri value='http://hl7.org/fhir/workflow'/>
+    <name value='Workflow Pattern'/>
+  </mapping>
+  <mapping>
+    <identity value='sct-concept'/>
+    <uri value='http://snomed.info/conceptdomain'/>
+    <name value='SNOMED CT Concept Domain Binding'/>
+  </mapping>
+  <mapping>
+    <identity value='v2'/>
+    <uri value='http://hl7.org/v2'/>
+    <name value='HL7 v2 Mapping'/>
+  </mapping>
+  <mapping>
+    <identity value='rim'/>
+    <uri value='http://hl7.org/v3'/>
+    <name value='RIM Mapping'/>
+  </mapping>
+  <mapping>
+    <identity value='w5'/>
+    <uri value='http://hl7.org/fhir/fivews'/>
+    <name value='FiveWs Pattern Mapping'/>
+  </mapping>
+  <mapping>
+    <identity value='sct-attr'/>
+    <uri value='http://snomed.org/attributebinding'/>
+    <name value='SNOMED CT Attribute Binding'/>
+  </mapping>
+  <kind value='resource'/>
+  <abstract value='false'/>
+  <type value='Observation'/>
+  <baseDefinition value='http://fhir.de/StructureDefinition/observation-de-vitalsign'/>
+  <derivation value='constraint'/>
+  <differential>
+    <element id='Observation.code.coding'>
+      <path value='Observation.code.coding'/>
+      <slicing>
+        <discriminator>
+          <type value='pattern'/>
+          <path value='$this'/>
+        </discriminator>
+        <rules value='open'/>
+      </slicing>
+      <min value='1'/>
+    </element>
+    <element id='Observation.code.coding:loinc'>
+      <min value='1'/>
+      <path value='Observation.code.coding'/>
+      <sliceName value='loinc'/>
+      <max value='1'/>
+      <patternCoding>
+        <system value='http://loinc.org'/>
+        <code value='9269-2'/>
+        <display value='Glasgow coma score total'/>
+      </patternCoding>
+    </element>
+    <element id='Observation.value[x]'>
+      <path value='Observation.value[x]'/>
+      <type>
+        <code value='Quantity'/>
+      </type>
+    </element>
+    <element id='Observation.value[x].unit'>
+      <path value='Observation.value[x].unit'/>
+      <patternString value='{score}'/>
```

Comment:
.code sollte das Pattern enthalten nicht .unit. Unit kann offen bleiben.
Das Pattern gerne auch auf .value hochziehen, dann sieht man alle Pattern in einem.

Reply:
Done!